noetic/distribution.yaml

```diff
@@ -5424,6 +5424,23 @@ repositories:
       url: https://github.com/swri-robotics/mapviz.git
       version: master
     status: developed
+  marine_msgs:
+    doc:
+      type: git
+      url: https://github.com/apl-ocean-engineering/marine_msgs.git
+      version: main
+    release:
+      packages:
+      - marine_acoustic_msgs
+      - marine_sensor_msgs
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/CCOMJHC/marine_msgs-release.git
```

Comment:
I'm not familiar with the release process, but seems weird to have both a CCOMJHC and an apl-ocean-engineering repo listed here. Was this line an oversight?